babel.config.js

```diff
@@ -1,72 +0,0 @@
-module.exports = function(api) {
-  var validEnv = ['development', 'test', 'production']
```

Comment:
This file was added when we webpacker-ified Argo, and we longer use webpacker for assets.